sdk/spring/spring-integration-azure-core/src/main/java/com/azure/spring/integration/core/handler/DefaultMessageHandler.java

```diff
@@ -48,7 +48,7 @@ public class DefaultMessageHandler extends AbstractMessageProducingHandler {
     private final String destination;
     private final SendOperation sendOperation;
     private boolean sync = false;
-    private ListenableFutureCallback<Void> sendCallback;
+    private BiConsumer<Void, Throwable> sendCallback;
```

Comment:
Why change to BiConsumer?

Reply:
It should be updated, but I don't have a better solution, any suggestions are welcome.